esp/bindings/http/platform/httpservice.cpp

```diff
@@ -73,7 +73,12 @@ CEspHttpServer::~CEspHttpServer()
                 if (paramStr && *paramStr)
                     logStr.appendf("?%s", paramStr);
 
-                DBGLOG("Request[%s]", logStr.str());
+                if (ctx->queryHasException())
+                    DBGLOG("Log request due to exception. Request[%s]", logStr.str());
```

Comment:
Perhaps simplify slightly to "Request with exception: Request[%s]" and "Request with long processing time: %u seconds. Request[%s]"?

Unless you think its important to be able to search or parse for "Log Request"?

Reply:
@afishbeck do you mean "Request with exception. Request[%s]"?